docs/outputs.rst

```diff
@@ -150,8 +150,8 @@ in precise geographic correspondence, with varying mesh densities: 32k (default)
 **Grayordinates files**.
 CIFTI is a container format that holds both volumetric (regularly sampled in a grid)
 and surface (sampled on a triangular mesh) samples.
-Sub-cortical time series are volumetric (supported spaces: ``MNI152NLin6Asym``), while cortical
-time series are sampled to surface (supported spaces: ``fsLR``).
+Sub-cortical time series are volumetric (space: ``MNI152NLin6Asym``), while cortical
```

Comment:
Since this is specified by HCP, I think adding the spaces names only clutters:
```suggestion
Sub-cortical time series are sampled on a regular grid derived from one MNI template, while cortical
```